app/src/operations/notification/index.ts

```diff
@@ -22,10 +22,10 @@ export default defineOperationApp({
 			type: 'string',
```

Comment:
Wondering if we should update this to `json`. Thoughts @azrikahar?

Reply:
100%! Purely an oversight on my part. #14021 updated a number of them from `string` to `json`, so it definitely makes sense to follow suit here.

Reply:
```suggestion
			type: 'json',
```